ci/cpu/build.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
```

Comment:
This file is called by the CPU-only builds that do the following:
1. Build conda packages and ensure they complete without error
2. Upload built conda packages if building from `master` branch; otherwise, disregard upload